djangobench/main.py

```diff
@@ -209,8 +216,7 @@ def insignificant(cls, text):
     def bad(cls, text):
         return cls.colorize(cls.BAD, text)
 
-
-def format_benchmark_result(result, num_points):
+def format_benchmark_result(result, num_points, control_data, experiment_data, show_median):
```

Comment:
I'm wondering whether `show_median` should be a keyword argument here as well - that list is getting a bit long but it's not too bad.